### PR TITLE
WT-5458 Fix Evergreen timeout failures in linux-directio test

### PR DESCRIPTION
**Pull request by tammybailey**
Here we update the `linux-directio` Evergreen test to use the `format.sh` script rather than calling `t` in a loop. This allows us to run tests in parallel, virtually ensuring that we will accumulate completed test results even if we generate long-running configs. We set the `format.sh` timeout to just under the two hour default Evergreen timeout. We also set the build distros to higher spec machines for this task. Successful patch builds below.

- [Patch One](https://evergreen.mongodb.com/version/5e28e34be3c3314abdf82885)
- [Patch Two](https://evergreen.mongodb.com/version/5e28e35857e85a429c58768e)
- [Patch Three](https://evergreen.mongodb.com/version/5e28e3631e2d176633e47276)

### CODE REVIEW

File: test/evergreen.yml [hunk 1]
```diff
@@ -1637,18 +1637,20 @@ tasks:
...
     commands:
       - func: "fetch artifacts"
       - func: "compile wiredtiger"
-      - func: "format test"
+      - func: "format test script"
         vars:
-          times: 3
-          config: ../../../test/format/CONFIG.stress
-          extra_args: direct_io=1
+          format_test_script_args: -t 110 -j 4 direct_io=1
```
Comment (lukech) [inline test/evergreen.yml]: I believe the `-j 4` part has been covered by the `format test script` function, specifically, the `${smp_command|}` argument of the `format.sh` call.

Comment (raviprakashgiri29) [reply test/evergreen.yml]: You can remove the "-j 4" option as it is already taken care of by ${smp_command|}.

**Comment by raviprakashgiri29** [comment]
Changes LGTM except for minor comments.

File: test/evergreen.yml (hunk 1 above)
Comment (tammybailey) [reply test/evergreen.yml]: The `smp_command` will use the number of processors on the machine as the number of threads to run, and that is not the intent here.

Comment (raviprakashgiri29) [reply test/evergreen.yml]: smp_command(number of processor from /proc/cpuinfo) is anyways getting used in the "format test script" implementation. 

Copied the below command from your patch build. 
**./format.sh -j $(grep -c ^processor /proc/cpuinfo) -t 110 -j 4 direct_io=1 2>&1** 

So, you don't need to specify it again.

Comment (tammybailey) [reply test/evergreen.yml]: I understand, but I do not want to set it to the number of processors on the machine. I want it set to a lower number.

**Pull request merged**